src/compiler/tsc.ts

```diff
@@ -675,7 +677,22 @@ namespace ts {
             usageText += getParamType(option);
 
             usageColumn.push(usageText);
-            descriptionColumn.push(getDiagnosticText(option.description));
+            let description: string;
+
+            if (option.name === "lib") {

```

Comment:
I don't want to apply this to all the list type commandline option because some of them containing special note such as the `--target` will indicate that es3 is a default one. I will add comment about this